system-tests/test/multi_domain_retries_spec.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path'
+import systemTests, { expect } from '../lib/system-tests'
+import Fixtures from '../lib/fixtures'
+
+const e2ePath = Fixtures.projectPath('e2e')
+
+const PORT = 3500
+const onServer = function (app) {
+  app.get('/multi_domain_secondary.html', (_, res) => {
+    res.sendFile(path.join(e2ePath, `multi_domain_secondary.html`))
+  })
+}
+
+describe('e2e cy.origin retries', () => {
+  systemTests.setup({
+    servers: [{
+      port: 4466,
+      onServer,
+    }],
+    settings: {
+      hosts: {
+        '*.foobar.com': '127.0.0.1',
+      },
+    },
+  })
+
+  systemTests.it('Appropriately displays test retry errors without other side effects', {
+    // keep the port the same to prevent issues with the snapshot
+    port: PORT,
+    spec: 'multi_domain_retries_spec.ts',
+    snapshot: true,
+    expectedExitCode: 1,
+    config: {
+      experimentalSessionAndOrigin: true,
+      retries: 2,
+    },
+    async onRun (exec) {
+      const res = await exec()
+
+      // verify that retrying tests with cy.origin doesn't cause serialization problems to spec bridges on test:before:run:async
+      expect(res.stdout).to.not.contain('TypeError')
+      expect(res.stdout).to.not.contain('Cannot set property message of  which has only a getter')
```

Comment:
The formatting on this seems off. Should we split this into two? Otherwise I imagine that stdout will never contain this exact phrase.

Reply:
Seems like [that was the error](https://github.com/cypress-io/cypress/pull/20628), but I agree seems specious to rely on white-space when it could differ between browsers.

Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/992d629d3008821d83187131c871f7e70cfbe94e